src/render.js

```diff
@@ -13,13 +13,16 @@ import options from './options';
  */
 export function render(vnode, parentDom, replaceNode) {
 	if (options._root) options._root(vnode, parentDom);
-	let oldVNode = parentDom._children;
+	let oldVNode = replaceNode ?
+		replaceNode._children || parentDom._children :
```

Comment:
This `||` is needed for the case when we first did `render(x, y, undefined)` and transition to a render with replaceNode. The second is needed for when we use replaceNode twice in a row.

Reply:
What about this?

```suggestion
replaceNode && replaceNode._children || parentDom._children
```